### PR TITLE
Fix problems with the response types

### PR DESCRIPTION
**Pull request by mlopes**

### CODE REVIEW

File: ncow-api/src/main/scala/net/ncow/ncow/api/NcowService.scala [hunk 1]
```diff
@@ -1,34 +1,33 @@
...
-import com.lightbend.lagom.scaladsl.api.transport.Method
 
 trait NcowService extends Service {
 
-  def registerBot(botRegistrationRequest: BotRegistrationRequest): ServiceCall[BotRegistrationRequest, BotRegistrationResponse]
+  def registerBot: ServiceCall[RegisterBotRequest, RegisterBotResponse]
 
   override final def descriptor = {
     import Service._
+    // @formatter:off
```
Comment (MarcusHayward) [inline ncow-api/src/main/scala/net/ncow/ncow/api/NcowService.scala:14]: Do we know what these do?

Comment (mlopes) [reply ncow-api/src/main/scala/net/ncow/ncow/api/NcowService.scala:14]: I think it prevents the editor from screwing up with the indentation.

**Comment by mlopes** [comment]
Added an implementation for the request handler. Still no command handler though:

screenshot

**Comment by MarcusHayward** [comment]
Amazing
Thanks Marco

**Pull request merged**